### PR TITLE
update search box examples

### PR DESCRIPTION
**Pull request by leexgh**
Fix: https://github.com/genome-nexus/genome-nexus/issues/505

### CODE REVIEW

File: src/component/SearchBox.tsx [hunk 1]
```diff
@@ -22,71 +20,6 @@ export type ExampleData = {
     label: string;
 };
 
-export const EXAMPLES: JSX.Element = (
```
Comment (leexgh) [inline src/component/SearchBox.tsx:25]: We don't use this anywhere

**Comment by leexgh** [comment]
@inodb Do we need to update grch38 examples?

**Comment by inodb** [comment]
@leexgh we should but not before we fix the data (do the examples work for grch38?)

**Comment by inodb** [comment]
@leexgh this issue also applies for the V600E example btw: https://github.com/genome-nexus/genome-nexus/issues/512

**Pull request merged**